src/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue'
+
+declare global {
+  namespace JSX {
+    // tslint:disable no-empty-interface
+    interface Element extends VNode {}
+    // tslint:disable no-empty-interface
+    interface ElementClass extends Vue {}
+    interface IntrinsicElements {
+      [elem: string]: any
+    }
+  }
+}
```

Comment:
Is this file needed? If not, please remove it.

Also for src/shims-vue.d.ts.

Also for src/utils/sleep.ts.

Reply:
Yes. shims-vue to avoid vs code lint error for .vue file and sleep is been used in unit test